source/features/more-dropdown.tsx

```diff
@@ -109,10 +109,7 @@ async function init(): Promise<void> {
 	);
 
 	// Selector only affects desktop navigation
-	for (const tab of select.all(`
-		.hx_reponav a[data-selected-links~="pulse"],
-		.hx_reponav a[data-selected-links~="security"]
-	`)) {
+	for (const tab of select.all('.hx_reponav a:is([data-selected-links~="pulse"], [data-selected-links~="security"])')) {
```

Comment:
What about this one? I think this looks ok?

Reply:
I don’t think it’s worth the unreadability of the line to avoid 9 repeated characters

Reply:
ok